pycape/function_ref.py

```diff
@@ -20,33 +20,73 @@
 
     assert fref == fref2
 """
+import enum
+from typing import Optional
+
+
+class FunctionAuthType(enum.Enum):
+    AUTH0 = 1
+    FUNCTION_TOKEN = 2
```

Comment:
`FUNCTION_TOKEN` should be renamed to `TOKEN` since in FunctionRef, you set the auth type with `self.set_auth_type(FunctionAuthType.TOKEN)`